lib/guardian_db/token.ex

```diff
@@ -5,19 +5,15 @@ defmodule GuardianDb.Token do
 
   use Ecto.Schema
   import Ecto.Changeset
-  import Ecto.Query, only: [from: 2]
+  import Ecto.Query, only: [where: 2]
```

Comment:
If you change this to:

```elixir
import Ecto.Query, only: [where: 3]
```

We can remove the `alias Ecto.Query` by changing line 60 as such:

```elixir
  def purge_expired_tokens! do
    timestamp = Guardian.timestamp()

    query_schema()
    |> where([token], token.exp < ^timestamp)
    |> GuardianDb.repo.delete_all()
  end
```